enterprise-suite/Makefile

```diff
@@ -90,53 +83,13 @@ lint-allyaml: allyaml  ## Check that all.yaml is valid (Need Minikube running)
 clean::  ## Delete make artifacts
 	rm -f $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML)
 
-
-## These targets used for dev work
-#
-
-# Expose services using NodePorts.  Normally disabled but often used
-# with minikube.  We enable for debugging purposes.
-# (If desired, you can override with something like "make <target> EXPOSE_METHOD=LoadBalancer"
-# or "make <target> EXPOSE_METHOD=" to get the default unexposed behavior.)
-EXPOSE_SERVICES=--set exposeServices=NodePort
-ifneq ($(origin EXPOSE_METHOD), undefined)
-  ifeq ($(EXPOSE_METHOD),)
-    EXPOSE_SERVICES=
-  else
-    EXPOSE_SERVICES=--set exposeServices=$(EXPOSE_METHOD)
-  endif
-endif
-
 allyaml: $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML)  ## Build all.yaml file
 
 $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML): $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz
 	$(call banner)
 	mkdir -p $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)
 	helm --namespace=$(NAMESPACE) template $< $(EXPOSE_SERVICES) > $@
 
-TILLER_NAMESPACE := kube-system
-
-install-helm:  ## Install required helm components into cluster
```

Comment:
I still use this target.  Why remove it?

Reply:
These were all moved into tests/smoketest_minikube.sh or tests/smoketest_openshift.sh, because the commands are different based on where you are installing it. In fact you can just run tests/smoketest_minikube.sh now, no need to use the make target. Is that okay? Otherwise I can try to extract the helm setup for minikube into its own shell script, if you want to just invoke that.

Reply:
Other benefit is to break cyclic dependencies between makefile and our scripts (the makefile calling a script which calls the makefile).

Reply:
Yes, it's a good point. Personally I don't use anything except lbc.py. I set up helm just from the command line, with helm init. It's hard to cater for everyone, so we should draw the line somewhere of what we keep in here. For me it's customer facing stuff like lbc.py, the rest is just infrastructure for running our CI.